demo/docker-compose.yaml

```diff
@@ -2,8 +2,8 @@ version: '3.9'
 
 services:
   helix:
-    container_name: helix-alpha
-    image: registry.helix-dev.synadia.io/helix:tag-rc
+    container_name: helix
+    image: registry.helix-dev.synadia.io/helix:v0.0.1-rc.5
```

Comment:
Our upgrade instructions right now are `docker compose pull` and `docker compose up`

If we want to stick with that, we should leave it as `tag-rc` and then switch to `latest` once the beta is published .

If we switch to tagging an explicit version here, we should switch the upgrade instructions to be `git pull` then `docker compose up`